Khan/KaTeX
# Fix MathML output for ' and large operators with limits

Summary:
This diff extracts MathML builder tests out into a separate file and
updates them to use jest snapshot testing.  It updates the output of
prime ' from using identifier nodes <mi> to operator nodes <mo>.  It
also updates large operators with limits to use munderover instead
of msupsub.

Test Plan:
- make test w/o errors
- verify mathml snapshots contain the desired markup

Fixes: #204 and #198.
